packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -34,6 +34,11 @@ BuildArch: noarch
 %endif
 
 Requires: subscription-manager
+
+%if %{build_tracer}
+Requires: %{name}-tracer = %{version}-%{release}
```

Comment:
It feels weird to have a separate package, yet always install it.

What is the reasoning for always requiring tracer?

Reply:
The request is that tracer is always enabled - so I did the simple thing and installed it as a dep of host-tools. Do you think it would be more appropriate to combine -tracer RPM into host-tools itself?

Reply:
If this would be a fresh package, then I'd say yes. Given it's an existing one, your approach is fine.

I think what I (personally) find weird about that: as a user, I perceive tracer as an addon that is not required for general day to day use of a system as a katello client. As such I'd expect tracer not to be installed by default (or at least removable w/o removing all of kht). But it might be just me being me and disliking to hoard code on my systems that I don't use.

@ehelms @ekohl thoughts?

BTW, is there a redmine that could be referenced?

Reply:
If I had to a pick a concern it would be the additional requests to the server on package transactions if the user isn't interested in the tracer feature. @sean797 just want to check with you: 'enable tracer by default' means 'always installed and enabled along with katello-host-tools', right?

Reply:
@jturel I think so, but I didn't write that requirement. Personally, I could go either way if we should install/enable this by default. I also have concerns regarding unnecessary client side execution, but I also want to advertise this feature because I think it's unique and very useful. Would using a weak dependency help?

Another route could be something like `katello-host-tools` includes tracer, but `katello-host-tools-minimal` does not include tracer?

Reply:
Weak deps are not a thing on EL7 and older, so while "yes, this is the correct approach", it wouldn't work for everyone.